tests/testapp/migrations/0003_alter_mytestpage_body_and_more.py

```diff
@@ -0,0 +1,313 @@
+# Generated by Django 4.0.10 on 2023-05-16 04:25
```

Comment:
suggestion: regenerate the testapp migrations to have just the initial one

Reply:
Done in https://github.com/wagtail/wagtail-factories/pull/74/commits/83a6015322ecd67e032b210f85abf3743d7d26e2